README.rst

```diff
@@ -1,97 +1,118 @@
+✅ Installation
+---------------
 
-Copyright 2006-2018 The Tahoe-LAFS Software Foundation
+For more detailed instructions, read `docs/INSTALL.rst <https://github.com/tahoe-lafs/tahoe-lafs/blob/master/docs/INSTALL.rst>`__ .
 
-You may use this package under the GNU General Public License, version 2 or,
-at your option, any later version. You may use this package under the
-Transitive Grace Period Public Licence, version 1.0, or at your option, any
-later version. (You may choose to use this package under the terms of either
-licence, at your option.) See the file `COPYING.GPL`_ for the terms of the
-GNU General Public License, version 2. See the file `COPYING.TGPPL`_ for
-the terms of the Transitive Grace Period Public Licence, version 1.0.
+-  Building Tahoe-LAFS on Windows: https://github.com/tahoe-lafs/tahoe-lafs/blob/master/docs/windows.rst
 
-See `TGPPL.PDF`_ for why the TGPPL exists, graphically illustrated on three
-slides.
+-  | OS-X Packaging:
+      | https://github.com/tahoe-lafs/tahoe-lafs/blob/master/docs/OS-X.rst
 
-.. _OSPackages: https://tahoe-lafs.org/trac/tahoe-lafs/wiki/OSPackages
-.. _Mac: docs/OS-X.rst
-.. _pip: https://pip.pypa.io/en/stable/installing/
-.. _COPYING.GPL: https://github.com/tahoe-lafs/tahoe-lafs/blob/master/COPYING.GPL
-.. _COPYING.TGPPL: https://github.com/tahoe-lafs/tahoe-lafs/blob/master/COPYING.TGPPL.rst
-.. _TGPPL.PDF: https://tahoe-lafs.org/~zooko/tgppl.pdf
+Once tahoe --version works, see `docs/running.rst <https://github.com/tahoe-lafs/tahoe-lafs/blob/master/docs/running.rst>`__ to learn how to set up your first Tahoe-LAFS node.
 
-----
+🤗 Contributing
+---------------
 
+As a community-driven open source project, Tahoe-LAFS welcomes contributions of any form:
+
+-  `Code patches <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/Patches>`__
+
+-  `Documentation improvements <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/Doc>`__
+
+-  `Bug reports <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/HowToReportABug>`__
+
+-  `Patch reviews <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/PatchReviewProcess>`__
+
+Before authoring or reviewing a patch, please familiarize yourself with the `Coding Standard <https://tahoe-lafs.org/trac/tahoe-lafs/wiki/CodingStandards>`__ and the `Contributor Code of Conduct <https://github.com/tahoe-lafs/tahoe-lafs/blob/master/docs/CODE_OF_CONDUCT.md>`__.
+
+🤖 Issues
+---------
+
+Tahoe-LAFS uses the Trac instance to track `issues <https://www.tahoe-lafs.org/trac/tahoe-lafs/wiki/ViewTickets>`__. Please email jean-paul plus tahoe-lafs at leastauthority dot com for an account.
+
+📑 Documentation
+----------------
+
+You can find the full Tahoe-LAFS documentation at our `documentation site <http://tahoe-lafs.readthedocs.io/en/latest/>`__.
+
+💬 Community
+------------
+
+Get involved with the Tahoe-LAFS community:
+
+-  Chat with Tahoe-LAFS developers at #tahoe-lafs chat on irc.freenode.net or `Slack <https://join.slack.com/t/tahoe-lafs/shared_invite/zt-jqfj12r5-ZZ5z3RvHnubKVADpP~JINQ>`__.
```

Comment:
I think I remember a comment about how this invite link will expire.  Do you know if that's true?  It seems like we should resolve it if so.

Reply:
slack was not built to host a community channel. The "Never expiring" invites seem to become invalid whenever a new "never expiring" is created according to some complaints about the issue here and there. 
It seems like the proper way to handle invites is using something similar to [slackin](https://github.com/rauchg/slackin) but that would require some work.

Reply:
Okay.  I thought it might expire after a week or a month or something.  I think I'll try to raise the topic of non-Slack web-based chat for Tahoe-LAFS with the community and see if there's interest in some other platform, perhaps with better properties.  For now this link seems fine though, since it should keep working unless someone goes out of their way to break it and because Slack is what we have.